extensions/item-assets/README.md

```diff
@@ -42,6 +42,8 @@ However, it is recommended that the Asset Definition is a complete set of all as
 
 Other custom fields, or fields from other extensions may also be included in the Asset object.
 
+Any property that exists for a collection-level asset object must also exist in the correspond assets object in each item. If a collection's asset object contains properties that are not explicitly stated in the item's asset object then that property does not apply to the item's asset. Collection-level asset objects can describe any of the properties of an asset, but those assets properties and values must also reside in the item's asset object.
```

Comment:
```suggestion
Any property that exists for a collection-level asset object must also exist in the corresponding assets object in each item. If a collection's asset object contains properties that are not explicitly stated in the item's asset object then that property does not apply to the item's asset. Item asset objects at the collection-level can describe any of the properties of an asset, but those assets properties and values must also reside in the item's asset object. To consolidate item-level asset object properties in an API setting, consider storing the STAC items without the larger properties internally as 'invalid' STAC items, and merge in the desired properties at serving time from the collection-level.
```

* I thin the correspond must be corresponding?
* Collection-level assets is a separate extension. To avoid confusion, changed the wording.
* I like the note on the PR description regarding APIs. Added that, too.